maps/rift/levels/rift-06-surface3.dmm

```diff
@@ -21387,9 +21387,7 @@
 /turf/simulated/floor/plating,
 /area/exploration/pathfinder_office)
 "iio" = (
-/obj/machinery/door/firedoor/multi_tile{
-	dir = 2
-	},
+/obj/machinery/door/firedoor/multi_tile,
```

Comment:
?

Reply:
I don't think I edited any doors, no. I'll open the branch to take a look.